flyteplugins/go/tasks/pluginmachinery/flytek8s/k8s_resource_adds.go

```diff
@@ -17,6 +17,9 @@ import (
 
 const (
 	flyteExecutionURL = "FLYTE_EXECUTION_URL"
+
+	FlyteInternalWorkerNameEnvVarKey        = "_F_WN"  // "FLYTE_INTERNAL_WORKER_NAME"
```

Comment:
Nit:
I'd prefer to be verbose and explicit and specify the full `FLYTE_INTERNAL_WORKER_NAME` and `FLYTE_INTERNAL_DIST_ERROR_STRATEGY` here. 

Reasons:

* Many `FLYTE_INTERNAL_...` env vars are already defined explicitly below.
* Seeing an env var called `_F_WN` in a pod is very untelling.

I'd prefer to also use the full name for `Name: "_F_PN",` below which was added in a previous PR.

Reply:
@fg91  See https://github.com/flyteorg/flyte/issues/5665

Reply:
let's keep the names short. these aren't meant to be user-facing anyways right?  just contributor facing.

Reply:
Didn't see https://github.com/flyteorg/flyte/issues/5665. Ok for me 👍